static/docs/commands-reference/import-url.md

```diff
@@ -97,9 +97,13 @@ CLI tools for each service.
 
 When DVC inspects a DVC-file, its dependencies will be checked to see if any
 have changed. A changed dependency will appear in the `dvc status` report,
-indicating the need to reproduce this import stage. When DVC inspects an
-external dependency, it uses a method appropriate to that dependency to test its
-current status.
+indicating the need to reproduce this <abbr>import stage</abbr>. When DVC
```

Comment:
it's not doing this for locked stages, @efiop am I correct? is there a way to run the status for them?

Reply:
Right. I'm not sure why that entire paragraph is in there actually. Removed in 4ffe63a